webpack.config.js

```diff
@@ -1,36 +1,40 @@
+const path = require('path');
 const HtmlWebpackPlugin = require('html-webpack-plugin');
 const ImagesConfigWebpackPlugin = require('image-config-webpack-plugin');
 const JsConfigWebpackPlugin = require('js-config-webpack-plugin');
 const ScssConfigWebpackPlugin = require('scss-config-webpack-plugin');
-const path = require('path');
 
 module.exports = {
-  entry: ['@babel/polyfill', './src/index.js'],
```

Comment:
Why have you deleted this?
@babel/polyfill converts all the latest JS syntax found in '.src/index.js' to the supported version of the running environment.

Reply:
my bad!